tests/bsim/bluetooth/host/l2cap/stress/src/main.c

```diff
@@ -479,7 +479,7 @@ static void test_central_main(void)
 	}
 	LOG_DBG("All peripherals disconnected.");
 
-	LOG_DBG("Max segment pool usage: %u bufs", max_seg_allocated);
+	LOG_INF("Max segment pool usage: %u bufs", max_seg_allocated);
```

Comment:
What's the purpose of this change? (I'm not necessarily opposed to it, but it's not clear to me how it relates to the rest of this change)

Reply:
It's not super-related. But I had to bump up the segment pool for the new config. 
It's nice to know what amount was actually used. I moved it to _INF as the _DBG log level kinda floods the output (it's more of a _TRACE level in this test).